rest_framework/utils/field_mapping.py

```diff
@@ -203,6 +203,9 @@ def get_relation_kwargs(field_name, relation_info):
     if to_many:
         kwargs['many'] = True
 
+    if to_field:
+        kwargs['to_field'] = to_field
```

Comment:
It's being used in `serializers.py` with the check against `id`. If we determine that the check isn't actually needed, then I think we could get away with just using `slug_field` here.